app/views/search/index.html.erb

```diff
@@ -20,15 +20,25 @@
   <% end %> 
     <div class="row">
       <div class="col">
-        <div class="list-group">
-          <%if @results %>
-            <% @results.each do |result|%>
+        <div class="list-group" id="exact-results">
+          <%if @exact_results.present? %>
+            <% @exact_results.each do |result|%>
               <a href=<%=url_for(result)%> class="list-group-item list-group-item-action" >
                 <span class= "<%= result.icon_class %>"></span>
                 <%= result.to_string %></a>
             <% end %>
           <% end %>
         </div>
+        <%if @exact_results.present? and @more_results.present? %> More Results <%end %>
```

Comment:
Are there any scenarios where there are `more_results ` when there are no `exact_results`? in anyway I would like to remove the `if @exact_results.present?` because it's not needed, is it?

Reply:
There are scenarios (e.g 'Prof Dr' instead of 'Prof. Dr.''). The P.O. is fine with this behaviour